src/application/test/open-api.test.ts

```diff
@@ -11,19 +12,25 @@ const api = getAPI(openApiClient(new URL(BASE_URL)).call)
 const mock = new MockAdapter(axios)
 
 describe('handle error responses', () => {
+  beforeAll(() => {
```

Comment:
don't think you need to do this before every test?

Reply:
No, it's working fine. We use the same setup in integration tests